website/versioned_docs/version-23.x/JestObjectAPI.md

```diff
@@ -466,117 +435,38 @@ test('works too', () => {
 
 Returns the `jest` object for chaining.
 
-### `jest.retryTimes()`
-
-Runs failed tests n-times until they pass or until the max number of retries are exhausted. This only works with [jest-circus](https://github.com/facebook/jest/tree/master/packages/jest-circus)!
+### `jest.isolateModules(fn)`
```

Comment:
same comment about this being available in v24

Reply:
above? I'll remove it from here, but I don't see the other comment you're referring to?

Reply:
the UI is so slow on that diff that it probably failed to be sent. I've seen references to `jest.isolateModules ` in 22 and 23 docs

Reply:
[`aefe5f2` (#10691)](https://github.com/facebook/jest/pull/10691/commits/aefe5f29472d614a87ff976eeedb151ab6b18675)